Mathlib/GroupTheory/Perm/Cycle/Basic.lean

```diff
@@ -60,13 +42,14 @@ open BigOperators
 
 variable {ι α β : Type*}
 
-namespace Equiv.Perm
 
 /-! ### `SameCycle` -/
 
 
 section SameCycle
 
+namespace Equiv.Perm
+
```

Comment:
What's going on here?

Reply:
What do you mean?

Reply:
Why did the namespace move inside the section? The namespace is supposed to be outside, with sections inside

Reply:
Does closing a namespace close all introduction of variables?
It is not so clear in the documentation.

Reply:
Apparently, it does not. These `section`had the intent of isolating variables introduced here and there.

Reply:
I did however made the change.